packages/adapter-prisma/src/migrations.ts

```diff
@@ -37,19 +37,27 @@ async function withMigrate<T>(
   }
 }
 
-export async function runPrototypeMigrations(dbUrl: string, schema: string, schemaPath: string) {
+export async function runPrototypeMigrations(
+  dbUrl: string,
+  schema: string,
+  schemaPath: string,
+  shouldDropDatabase = false
```

Comment:
Note that I didn't add a changeset for this because there hasn't yet been a release where `runPrototypeMigrations` was publicly exported.